dojo/tools/anchore_grype/parser.py

```diff
@@ -23,44 +22,143 @@ def get_findings(self, file, test):
+                finding_references += f'**Vulnerability Datasource:** {vuln_datasource}\n'
+            if vuln_urls:
+                if len(vuln_urls) == 1:
+                    if vuln_urls[0] != vuln_datasource:
+                        finding_references += f'**Vulnerability URL:** {vuln_urls[0]}\n'
+                else:
+                    finding_references += '**Vulnerability URLs:**\n'
+                    for url in vuln_urls:
+                        if url != vuln_datasource:
+                            finding_references += f'- {url}\n'
+            if rel_datasource:
+                finding_references += f'**Related Vulnerability Datasource:** {rel_datasource}\n'
+            if rel_urls:
+                if len(rel_urls) == 1:
+                    if rel_urls[0] != vuln_datasource:
+                        finding_references += f'**Related Vulnerability URL:** {rel_urls[0]}\n'
+                else:
+                    finding_references += '**Related Vulnerability URLs:**\n'
+                    for url in rel_urls:
+                        if url != vuln_datasource:
+                            finding_references += f'- {url}\n'
+            if finding_references and finding_references[-1] == '\n':
+                finding_references = finding_references[:-1]
+
+            finding_cvss3_score = None
+            finding_cvss3 = None
+            if vuln_cvss:
+                finding_cvss3_score, finding_cvss3 = self.get_cvss(vuln_cvss)
+            if not finding_cvss3_score and rel_cvss:
+                finding_cvss3_score, finding_cvss3 = self.get_cvss(rel_cvss)
+
+            dupe_key = finding_title
             if dupe_key in dupes:
-                find = dupes[dupe_key]
-                find.nb_occurences += 1
+                finding = dupes[dupe_key]
+                finding.nb_occurences += 1
             else:
                 dupes[dupe_key] = Finding(
-                    title=f"Vulnerable component ({purl.name}:{purl.version}) {cve}",
-                    description=description,
-                    cve=cve,
-                    severity=severity,
-                    impact="No impact provided",
-                    mitigation="No mitigation provided",
-                    references="[{}](https://nvd.nist.gov/vuln/detail/{})".format(
-                        cve, cve
-                    ),
-                    static_finding=True,
-                    dynamic_finding=False,
-                    component_name=purl.name,
-                    component_version=purl.version,
-                    vuln_id_from_tool=cve,
-                    nb_occurences=1,
-                )
+                            title=finding_title,
+                            description=finding_description,
+                            cve=finding_cve,
+                            cwe=1352,
+                            cvssv3=finding_cvss3,
+                            cvssv3_score=finding_cvss3_score,
```

Comment:
If you provide the vector, the score is calculated automatically.
```suggestion
```
So better to provide the vector only.

Reply:
done